test/remote/file.go

```diff
@@ -0,0 +1,57 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package remote
+
+import (
+	"context"
+	"net/http"
+	"time"
+
+	"github.com/golang/glog"
+	"golang.org/x/oauth2/google"
+	filev1beta1 "google.golang.org/api/file/v1beta1"
+)
+
+func GetFileClient() (*filev1beta1.Service, error) {
+	const retries = 10
+	const backoff = time.Second * 6
```

Comment:
We can create global constants for this retry and backoff

Reply:
Done.